components/encumbered/gst-plugins-bad1/Makefile

```diff
@@ -99,7 +109,6 @@ REQUIRED_PACKAGES += library/audio/openal
 REQUIRED_PACKAGES += library/desktop/cairo
```

Comment:
REQUIRED_PACKAGES += library/audio/soundtouch

The soundtouch package is installed on the build server.
This will bring
file path=usr/lib/$(MACH64)/gstreamer-1.0/libgstsoundtouch.so
file path=usr/lib/gstreamer-1.0/libgstsoundtouch.so
to the manifests